docs/tutorials/sinks/1_basic_sink.md

```diff
@@ -328,13 +363,32 @@ sinks:
       - stdin
 ```
 
-This simply connects a `stdin` source to our `basic` sink. Run it with:
+This simply connects a `stdin` source to our `basic` sink. 
+
+## vdev
+
+Vector provides a build tool `vdev` that simplifies the task of building Vector. Install
+`vdev` using the instructions [here][vdev_install]. 
+
+With `vdev` installed we can run Vector using:
+
+```sh
+vdev run ./basic.yml
```

Comment:
```suggestion
cargo vdev run ./basic.yml
```

Right? Unless you've built it and put it into your path?

Reply:
I did [this](https://github.com/vectordotdev/vector/tree/master/vdev#installation)..

Reply:
Ah, yep - that works. I've been using `cargo vdev` pattern ever since Bruce added it, saves me from having to update the `.cargo/bin/vdev` binary whenever it changes - that might be a good reason to change that readme 🤔